src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
I agree that "copy" is closer to JES backend, although it's a big waste of copy time / storage space IMO. I think we should still try to optimize features in a backend-specific way as long as the functional behavior stays the same across all backends.

For the soft/hard my thinking was that if it works with soft why not go with that but I agree it might generate issues. We could get rid of it all together and directly go with hard.